english/hulk/smash.md

```diff
@@ -1,2 +1,3 @@
 You suddenly turn green and grow twice your size. 
-As Hulk, you easily break through the floor and fall, only to find that it's turtles all the way down.
\ No newline at end of file
+As Hulk, you easily break through the floor and fall, only to find that it's turtles all the way down.
+You flail your arms wildly, whacking the turtles left and right, screaming "Hulk Smash!" with every broken shell. You can't get them all though, and a particularly large ones knocks your head and you pass out.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.